### ISSUE TITLE
png plugin laser scaling doesn't work with the incident field laser 

### ISSUE THREAD

**Issue by pordyna** [ComputationalRadiationPhysics/picongpu#4158]
This is a minor issue, but can be somehow confusing. The option to normalize the png output based on the laser intensity set via `#define EM_FIELD_SCALE_CHANNEL1 1`   in `png.param` is based on the old laser implemementation. When someone uses the new incident field, has disabled the old laser, and combines it with this png setting, the compilation will fail with:

```
/home/ordyna35/src/picongpu-git/include/pmacc/../picongpu/unitless/png.unitless(55): error: no instance of function template "mpl_::assertion_failed" matches the argument list
            argument types are: (mpl_::failed ************(picongpu::You_can_not_scale_your_preview_to_laser_without_using_a_laser___change_png_param_________206::************)(pmacc::StaticAssertError))
```

Since now, we can have multiple lasers, it is not clear which intensity to use. So maybe we should  let the user define it again in `png.param`.

**Comment by sbastrakov** [comment]
True, there `laser` only means what is in `laser.param`. This should at least be documented more clearly.

It is in principle possible to add a new value for `EM_FIELD_SCALE_CHANNEL` defines to normalize to incident field. As you pointed out it's not clear what to use for multiple lasers enabled. But maybe we could just throw an error in case this setting is used and multiple lasers are enabled.

Regarding allowing a user to directly define an amplitude - I agree, we should also add it. It is unrelated to having a setting for incident fields.

**Comment by sbastrakov** [comment]
Implemented with #4172.
